modules/libcom/src/misc/epicsString.c

```diff
@@ -442,3 +442,11 @@ double epicsStrSimilarity(const char *A, const char *B)
     free(dist1);
     return ret;
 }
+
+/* Identical to strncpy but ensures dest is terminated */
+char* epicsStrnCpy(char* dest, const char* src, size_t destSize)
+{
+    char* ret = strncpy(dest, src, destSize);
+    dest[destSize-1] = 0;
+    return ret;
```

Comment:
You're right, this does look almost equivalent to `strlcpy` (aside from return value), so I'll remove this. Should I define a `epicsStrlCpy` in this branch or let that other PR handle it?

> I'm somewhat reluctant to go with defining our own str*() functions.

My plan here was to only provide a slightly safer alternative `strncpy` because of its odd, occasionally unsafe behavior. The rest of the C string library is a bit less error-prone, so defining `epicsStr*` wouldn't really be necessary.